src/languages/sass.coffee

```diff
@@ -2,7 +2,6 @@ module.exports = {
 
   name: "Sass"
   namespace: "sass"
-  fallback: ['css', 'scss']
```

Comment:
I would prefer to have `fallback: ['css']` remain (be put back) such that users that have already configured `css` will be able to immediately use `Sass` without any changes. Advanced `Sass` users will, of course, be able to edit configuration under the `sass` namespace which will take precedence over `css` namespace.